omegaconf/dictconfig.py

```diff
@@ -619,8 +620,8 @@ def _set_value_impl(
         elif _is_interpolation(value, strict_interpolation_validation=True):
             self.__dict__["_content"] = value
             self._metadata.object_type = None
-        elif value == "???":
-            self.__dict__["_content"] = "???"
+        elif _is_missing_value(value):
```

Comment:
Same question as above: is it fixing a bug?

Reply:
no. this is systematically getting rid of all "???" from the codebase.